### PR TITLE
Blog app - Creating data model

### PR DESCRIPTION
**Pull request by ArthurGC**
# Documentation: Blog app - Ruby on Rails

## :blue_book: Steps

- [x] Build my project schema.
- [x] Create and run the necessary migration files.
- [x] Table and column names match the ERD diagram.
- [x] All columns that are foreign keys have a corresponding index

### CODE REVIEW

File: db/migrate/20211104183055_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
```
Comment (Davidosky007) [inline db/migrate/20211104183055_create_users.rb:5]: * Kindly change `t.text: photo` to `t.string: photo` as the photo will be a link to an image so therefore the right type would be `string`

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @ArthurGC 👋, 

Good job so far! 💪 
* Well written PR ✔️ 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ArthurGC** [comment]
Hi 😉 
I created a migration to fix it. So I don't have anything to change.

https://github.com/ArthurGC/blog-app-project/blob/3dbec1fb47b2acfbb06aedf2904477753be2ea4d/db/migrate/20211104185607_change_data_type_of_photo.rb#L1-L5

The schema: 

https://github.com/ArthurGC/blog-app-project/blob/3dbec1fb47b2acfbb06aedf2904477753be2ea4d/db/schema.rb#L48-L55

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @ArthurGC 👋🏽
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**